mailgun/route_handlers.py

```diff
@@ -60,20 +59,20 @@ def handle_mailing_list_email_route(request):
         return JsonResponse({'error': message}, status=406)  # Return status 406 so Mailgun does not retry
 
     # Always include teaching staff addresses with members addresses, so that they can email any list in the course
-    teaching_staff_addresses = ml.teaching_staff_addresses
-    member_addresses = teaching_staff_addresses.union(
-                           [m['address'] for m in ml.members])
+    try:
+        teaching_staff_addresses = ml.teaching_staff_addresses
+    except Exception as e:
+        logger.exception("Failed to find teaching staff addresses")
```

Comment:
If you're intending to continue here, you need to do something like `teaching_staff_addresses = set()`, or the very next line will fail.